packages/runtime/src/helpers/edge.ts

```diff
@@ -358,11 +358,105 @@ export const writeEdgeFunctions = async ({
   await writeJSON(join(edgeFunctionRoot, 'edge-shared', 'nextConfig.json'), nextConfig)
   await copy(join(publish, 'prerender-manifest.json'), join(edgeFunctionRoot, 'edge-shared', 'prerender-manifest.json'))
 
+  // early return if edge is disabled
+  if (destr(process.env.NEXT_DISABLE_NETLIFY_EDGE)) {
+    return
```

Comment:
Do we need a log message here if we early out? Maybe to help with debugging?

Reply:
I've added a log message for clarity :)